pkg/get/tools.go

```diff
@@ -3810,6 +3810,30 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}_{{.VersionNumber}}_{{$osStr}}_{{$arch}}.{{$extStr}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "crossplane",
+			Repo:        "crossplane",
+			Name:        "crossplane",
+			Description: "Simplify some development and administration aspects of Crossplane.",
```

Comment:
Since versions appear to be tracked at https://github.com/crossplane/crossplane/releases should `GitHubVersionStrategy` be used here?

Reply:
If you need a reference point for how `GitHubVersionStrategy` works with an URL template then the hashicorp tools follow that pattern.